bitgener.d.ts

```diff
@@ -0,0 +1,63 @@
+export = bitgener;
```

Comment:
it might be better than publishing this type as a separate package.

Or contributing upstream can be an option.

Reply:
I talked to maintainer and It seems they want to keep their library totally pure javascript. so....

Reply:
Then, how about publishing on [DT](https://definitelytyped.github.io/)?

Reply:
Alright!